src/libp2p/collection.rs

```diff
@@ -377,8 +377,9 @@ where
 
     /// Adds a new multi-stream connection to the collection.
     ///
-    /// Note that no [`Event::HandshakeFinished`] event will be generated. The connection is
-    /// immediately considered as fully established.
+    /// Must be passed the moment (as a `TNow`) when the connection as been established, in order
+    /// to determine when the handshake timeout expires.
+    // TODO: add an is_initiator parameter? right now we're always implicitly the initiator
```

Comment:
if it's an incoming substream, we're not initiators

Reply:
This function is for adding multi-stream connections as a whole.

If one day we add support for browser-to-browser WebRTC connections, then `is_initiator` could be `false`.